com.unity.multiplayer.mlapi/Runtime/Metrics/NetworkMetrics.cs

```diff
@@ -128,6 +132,34 @@ public void TrackObjectDestroyReceived(ulong senderClientId, ulong networkObject
             m_ObjectDestroyReceivedEvent.Mark(new ObjectDestroyedEvent(new ConnectionInfo(senderClientId), new NetworkObjectIdentifier(gameObjectName, networkObjectId), bytesCount));
         }
 
+        public void TrackRpcSent(ulong receiverClientId, ulong networkObjectId, string rpcName, ulong bytesCount)
+        {
+            if (!m_NetworkGameObjects.ContainsKey(networkObjectId))
+            {
+                m_NetworkGameObjects[networkObjectId] = new NetworkObjectIdentifier($"NetworkGameObject_{networkObjectId}", networkObjectId);
```

Comment:
I'm confused about the name here. Why have a name if it's just hard coded to the network id?

Reply:
This looks slightly different to the other metrics, here we're updating two metrics, one is tracking network objects (which if this is something we should do I'll need to make changes to the network object spawned/destroyed metrics as these are just in their own metric type), and one is tracking the RPCs. In the other "Track" functions we're only updating one metric type.

Reply:
My idea here is that if for some reason we do not have the GameObject registered yet, we could create it on the fly and have it's name be the id. I don't mind removing that though.

Reply:
Since it allocates a runtime string I'd say it isn't worth it. I'd rather put a literal like "Unknown" for this case